keylime/cloud_verifier_tornado.py

```diff
@@ -278,7 +278,7 @@ def get(self):
             try:
                 agent = (
                     session.query(VerfierMain)
-                    .options(joinedload(VerfierMain.ima_policy))
+                    .options(noload(VerfierMain.ima_policy))
```

Comment:
This makes it so that the attribute is always None, right?

In this case it should be fine, but maybe add an comment either here or in the commit messages why noload is used.

Reply:
Added a comment explaining why it was **temporarily** set as `noload`